[plugin/test] Plugin test with jdk14 (#5352)
apache/skywalking: docs/en/guides/Plugin-test.md

## Environment Requirements

1. MacOS/Linux
2. JDK 8+
3. Docker
4. Docker Compose

## Case Base Image Introduction

> The test framework provides `JVM-container` and `Tomcat-container` base images including JDK8, JDK14. You could choose the suitable one for your test case, if either is suitable, **recommend choose `JVM-container`**.

Reviewer: ```suggestion
The test framework provides `JVM-container` and `Tomcat-container` base images including JDK8, JDK14. You could choose the suitable one for your test case, if both are suitable, **`JVM-container` is preferred**.
```
